src/api/utils.ts

```diff
@@ -48,3 +50,36 @@ export const getAxiosErrorMetadata = (error: AxiosError): ErrorMetadata => ({
 });
 
 export const stringifyUrl = (url: string, query: string) => `${url}?${query}`;
+
+/**
+ * Get the text of a field in a NeTeX entity in the correct language. If English
+ * is requested, it will fallback to Norwegian if no English text is found. If
+ * neither English nor Norwegian is found the first text in the provided texts
+ * array is returned.
+ */
+export const getTextInLanguage = (
+  texts: LanguageAndText[],
+  language: Language,
+) => {
+  if (language === Language.English) {
+    const englishText = texts.find((t) => t.lang === 'eng');
+    if (englishText) return englishText.value;
+  }
+  const norwegianText = texts.find((t) => t.lang === 'nor' || t.lang === 'nob');
+  if (norwegianText) return norwegianText.value;
+
+  return texts[0].value;
+};
+
+/**
+ * Wrapper for getting the name of a NeTeX entity in the given language.
+ */
+export const getNameInLanguage = <
```

Comment:
Kinda feel that the function name `getNameInLanguage` is not the best. I am happy for better sounding alternatives.

Reply:
They are reference-data related. So possibly `getReferenceDataName`?